docs/Readme.md

```diff
@@ -120,8 +120,8 @@ It is strongly advised to read the [application license](../LICENSE) as well.
 ## History
 
 At first, this project was a fork of the [Discord-Electron], but then eventually
-I had rewritten it as *Electron Discord Web App* project, which is currently
-called *WebCord*.
+I rewritten it as *Electron Discord Web App* project, which is now
```

Comment:
I don't think *I rewritten* is correct (and removing *have* is anything better)… I don't think it's the correct form, at least if the verb is not in *\* Perfect* Tense… I'm still learning English through and if you're native speaker, feel free to share your thoughts.

Reply:
```suggestion
I rewrote it as *Electron Discord Web App* project, which is now
```
I think that is the correct way, at least it sounds better to me.